pkg/cloud/coxedge/client.go

```diff
@@ -44,6 +45,13 @@ func NewClient(service, environment, apiKey string, httpClient *http.Client) (*C
 		client:  httpClient,
 	}
 
+	if len(organisationID) > 0 {
+		// Setting it to '?id' so we can just append it to the end of the url
+		client.organisationID = "?" + organisationID[0]
```

Comment:
Can you do the formatting into the query at the url construction, instead of doing implicitly here.